### ISSUE TITLE
Refactor URL Helpers Module for Readability and Consistency"

### ISSUE THREAD

**Issue by am-nimrah** [sheharyarn/cloudup.dev#27]
Refactored URL Helpers module for improved readability and consistency, utilizing url.endsWith("/") and simplifying the removeTrailingSlash function. Cleaned up formatting and naming for a more cohesive code structure.

**Comment by ahmadzahiddd** [comment]
What I Liked:

Acknowledging the need for refactorings to enhance readability and consistency in the URL Helpers Module.
Suggestions/Concerns:

Consider breaking down complex functions into smaller, more modular ones for better readability.
Ensure consistent naming conventions across the module for a uniform codebase.
Document the purpose and usage of each helper function to aid future developers.
Verify if the module adheres to any established coding standards or best practices.
Reproducible Steps (if applicable):

Share specific examples of URLs and helper functions that illustrate readability or consistency concerns.
Additional Context:

Collaborate with team members to gather diverse perspectives on potential improvements.
Conduct code reviews to ensure the changes align with the overall architecture.
Gratitude:

Thank you for taking the initiative to refine the URL Helpers Module. Your efforts will contribute to a more maintainable and understandable codebase.

**Comment by sheharyarn** [comment]
I agree with the suggestion, I've linked this issue to your PR. This will be automatically closed once your PR is merged.

Thank you! 🌟
